srcpkgs/sublime-text3-bin/template

```diff
@@ -0,0 +1,55 @@
+# Template file for 'sublime-text3-bin'
+pkgname=sublime-text3-bin
+version=3211
+revision=1
```

Comment:
not quite sure how the version/revision should be done here and on the metapackage. open for advice.